src/js/view/adskipbutton.js

```diff
@@ -2,16 +2,19 @@ define([
     'utils/helpers',
     'utils/css',
     'events/events',
+    'utils/ui',
     'utils/backbone.events',
     'utils/underscore',
     'handlebars-loader!templates/adskipbutton.html'
-], function(utils, cssUtils, events, Events, _, AdSkipTemplate) {
+], function(utils, cssUtils, events, UI, Events, _, AdSkipTemplate) {
 
     var AdSkipButton = function(skipMessageCountdown, skipMessage) {
         this.skipMessage = skipMessage;
         this.skipMessageCountdown = skipMessageCountdown;
         this.skipMessage = skipMessage;
 
+        new UI (_instreamSkipContainer).on(events.touchEvents.CLICK, skipAd).on(events.touchEvents.TAP, skipAd);
```

Comment:
You can listen to both event at once using 'click tap'. The problem with this approach of registering two event listeners is that some devices will dispatch both events. Instead you want to test for the presence of the Touch class and if present listen for tap else listen for click.

> On May 29, 2015, at 6:54 PM, Donato Borrello <notifications@github.com> wrote:
> 
> In src/js/view/adskipbutton.js:
> 
> >  
> >      var AdSkipButton = function(skipMessageCountdown, skipMessage) {
> >          this.skipMessage = skipMessage;
> >          this.skipMessageCountdown = skipMessageCountdown;
> >          this.skipMessage = skipMessage;
> >  
> > +        new UI (_instreamSkipContainer).on(events.touchEvents.CLICK, skipAd).on(events.touchEvents.TAP, skipAd);
> This pattern can be a little confusing, an extra set of parenthesis would disambiguate intention
> 
> (new UI(..)).on(..)
> —
> Reply to this email directly or view it on GitHub.
>